### ISSUE TITLE
Problems with MRBS Joomla Authentication and Joomla "Authentication - LDAP" plugin

### ISSUE THREAD

**Issue by Chrivi123** [meeting-room-booking-system/mrbs-code#3521]
**Describe the problem you're facing/question you have:**
We are currently encountering issues with Joomla Authentication when combined with the Joomla "Authentication - LDAP" plugin. 
Whenever this plugin is active, our MRBS login functionality ceases to function.

**Browser details (please complete the following information):**
 - OS: Windows
 - Browser & Version: Google Chrome,  Firefox, Edge
 - Version: all browsers are on version 117

**Server details (please complete the following information):**
 - MRBS version 1.11.3.
 - Web server: Apache/2.4.53 (AlmaLinux)
 - OS: AlmaLinux release 9.2 
 - PHP version: PHP 8.1.23
 - Database used: 10.5.16-MariaDB

**Additional context**
To test this, we set up Joomla version 4.3.4 and MRBS version 1.11.3. 

In Joomla, we created the following:
- Groups:
  -- "MRBS Admin" group (under Group Parent Manager).
  -- "MRBS User" group (under Group Parent Registered).
- Users:
  -- "admin1" user (assigned to groups "MRBS Admin" & "Registered").
  -- "user1" user (assigned to groups "MRBS User" & "Registered").
- Access Levels:
  -- "MRBS Admin Level" with ID 7 for Group "MRBS Admin".
  -- "MRBS User Level" with ID 8 for Group "MRBS User".

Here is the $authentication section from config.inc.php:
$auth["type"] = "joomla";
$auth["session"] = "joomla"; 
$auth['joomla']['rel_path'] = '..';  
$auth['joomla']['admin_access_levels'] = array(7); 
$auth['joomla']['user_access_levels'] = array(8); 

As mentioned previously, the login works as expected when the Authentication - LDAP plugin is disabled. 
However, when it is enabled, we encounter an "Unknown User" error.

It's important to mention that currently, even with the "Authentication - LDAP" plugin enabled, we can successfully log in to our Joomla site (using LDAP or local Joomla user credentials) and then navigate to the MRBS link while maintaining the login session. 
However, when we attempt to log in via the MRBS login button, it fails to work as soon as the LDAP Plugin is enabled.

We would greatly appreciate any assistance on this problem.

**Comment by campbell-m** [comment]
Can you give a link to the plugin that you are using?  Also, which version of Joomla! are you using?

**Comment by Chrivi123** [comment]
We are using Joomla version 4.3.4 and it's the native Joomla "Authentication - LDAP" plugin.
It is disabled by default, but we have it enabled to authenticate our users.

**Comment by Chrivi123** [comment]
Hi, @campbell-m , were you able to reproduce the problem?

**Comment by campbell-m** [comment]
Sorry, I haven't had a chance yet.  I'll try and take a look later this week.

In the meantime are you just authenticating your MRBS users direct to LDAP?

**Comment by campbell-m** [comment]
Yes, I've now managed to reproduce the problem.   However finding the root cause may be more challenging ...

**Comment by campbell-m** [comment]
Now fixed in [8a7a8fa](https://github.com/meeting-room-booking-system/mrbs-code/commit/8a7a8fac1e4bf144e05119f7a56b149584c9d733).

**Comment by campbell-m** [comment]
I've now committed a better fix in [80f4640](https://github.com/meeting-room-booking-system/mrbs-code/commit/80f4640b072b3060787db4309219db8791db5213).
